Criacao da Guia de contribuicao na wiki

marcusmartinss opened the pull request:
Authored by: Marcus Martins <marcusmartinsxx@users.noreply.github.com>

Co-authored-by: Victor-oss <Victor-oss@users.noreply.github.com>

marcusmartinss committed:
Criacao da Guia de contribuicao na wiki

lievertom commented on .github/CONTRIBUTION.md:
```diff
@@ -0,0 +1,110 @@
+# Guia de Contribuição
+
+Antes de realizar uma contribuição, conheça um pouco sobre o Feudo Laranja Ave do Paraíso na nossa [documentação](https://feudo-laranja-ave-do-paraiso-ds-2021-2.github.io/wiki/management/methodology-plan/).
```
pode redirecionar pra home da wiki: https://feudo-laranja-ave-do-paraiso-ds-2021-2.github.io/wiki/

marcusmartinss replied:
Eu tinha pensado nisso, ficamos em duvida na hora mas decidimos seguir o que estava no texto.

lievertom commented on docs/project/CONTRIBUTION.md:
```diff
@@ -0,0 +1,116 @@
+# Guia de Contribuição
+
+Antes de realizar uma contribuição, conheça um pouco sobre o Feudo Laranja Ave do Paraíso na nossa [documentação](https://feudo-laranja-ave-do-paraiso-ds-2021-2.github.io/wiki/management/methodology-plan/).
```
usar o link para home: https://feudo-laranja-ave-do-paraiso-ds-2021-2.github.io/wiki/

lievertom reviewed (changes requested):
Falta referenciar o guia de conduta no arquivo mkdocs.yml

Victor-oss committed:
Co-authored by: Marcus Martins <marcusmartinsxx@users.noreply.github.com>

Victor-oss commented:
Mudanças realizadas

lievertom commented on mkdocs.yml:
```diff
@@ -32,6 +32,7 @@ nav:
     - Plano metodológico: "management/methodology-plan.md"
     - Plano de comunicação: "management/communication-plan.md"
     - Quadro de disponibilidade: "management/availability-board.md"
+    - Guia de Conduta: "https://feudo-laranja-ave-do-paraiso-ds-2021-2.github.io/wiki/"
```
O guia de contribuição, não guia de contuda, deve ficar na aba projeto e deve ser feita a referencia para a localização do documento:

```
    nav:
        - Ger ....
        - Projeto:
            -  Guia de contribuição: "project/CONTRIBUTING.md"
```

marcusmartinss replied:
Feito!

lievertom replied:
faltou adicionar a aba projeto e nela adicionar o guia de contribuição

lievertom reviewed (changes requested):
O nome dos arquivos deve  ser CONTRIBUTING.md

marcusmartinss committed:
Atualização do mkdocks.yml

Correção de alteração do nome do Guia de contribuição na barra de navegação, e utilização do arquivo onde estão os dados de contribuição.

marcusmartinss committed:
Correção na barra de navegação

Adicionada a aba de projetos e, dentro desta, a aba comunidade com o link para a contribuição no projeto.

Victor-oss committed:
Merge branch 'main' into criacao-guia-de-contribuicao

lievertom reviewed (approved):
:+1:
